docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,76 @@
-# L1 / L2 Interoperability
+# L1 / L2 interoperability
 
-While most of the execution will happen on L2, some use cases require interoperability with the L1 chain. The main use cases are building complex bridges, maintaining governance smart contracts on one chain that govern contracts on other chains, etc.
+## Common use cases
 
-In addition, the L2 censorship resistance is derived from the underlying chain, so the ability to send messages from Ethereum to zkSync is an important part of the censorship-resistance mechanism called the [priority queue](#priority-queue).
+Many use cases require multi-layer interoperability, such as:
 
-Sending transactions from Ethereum to zkSync is done via the zkSync smart contract. It allows the sender to request transactions directly from L1. Thereby allowing the permissionless passing of any data from Ethereum into zkSync.
-[Read more](../bridging/l1-l2.md) about messaging from L1 to L2.
+- The network's censorship resistance.
+- Custom bridges.
+- Multi-layer governing smart contracts.
+- Multi-layer transfers.
 
-## Priority queue
+## L1 to L2 communication
 
-The goal of the priority queue is to provide a censorship-resistant way to interact with zkSync in case the operator becomes malicious or unavailable.
-The way the priority queue works in zkSync Era is very close to how it worked in the previous version of zkSync.
-For the full picture, we first present how the priority queue works on zkSync Lite.
-This gives the rationale for the new design of the priority queue for zkSync Era.
+L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/matter-labs/v2-testnet-contracts/blob/b8449bf9c819098cc8bfee0549ff5094456be51d/l1/contracts/zksync/interfaces/IZkSync.sol#L4) inherited interfaces.
 
-### How it works in zkSync Lite
+:::tip
+- If you prefer to learn-by-doing, the [cross chain governance tutorial](../../tutorials/cross-chain-tutorial.md) is a practical example of layer interoperability.
+:::
 
-In the previous version of zkSync, we only had two operations that could be sent to zkSync from L1:
+### Gas estimation
 
-- `Deposit` to bridge funds from Ethereum to zkSync.
-- `FullExit` to bridges the funds back from Ethereum (this is essentially the same as `Withdraw` in zkSync Era).
+The system processes gas estimation for transactions implicitly. However, it is also possible to implement the gas estimation processes explicitly.
+
+:::tip L1 to L2 gas estimation for transactions
+- Gas is measured in amount and not ERC20 token value so actual costs can vary.
```

Comment:
I don't get this comment

Reply:
Amount of gas versus value of gas, i.e. 10 beans ... instead of £1.50 per bean = £15.

Reply:
1. why do we even mention erc20 token value?
2. as I wrote in the subsequent comment, this is just how Ethereum works. Not sure why we need to explicitly state it.

Reply:
Because I was not clear whether it was value or amount - as the reader. And I expect a newbie would be the same.

Reply:
ok, I just find "ERC20 token value" kinda confusing, but if you want to stress it, I don't have a better suggestion (we can use "monetary value" but I'm not sure it's better)

Reply:
let's put it on the back-burner and if we come up with something better... we'll add it wdyt?